catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/handlers/CqlTransformHandler.java

```diff
@@ -217,4 +236,46 @@ private void attachFileToResponse(
         "Successfully output file using transformer id {}",
         queryResponseTransformer.getProperty("id"));
   }
+
+  private Map<String, Serializable> cswTransformArgumentsAdapter() {
```

Comment:
✏️ This is very CSV-specific, probably best to pull it out into it's own class.

Reply:
Good point; might be worthwhile to inject adapters into this class via blueprint